src/lib/gundb/UserStorageClass.js

```diff
@@ -1160,10 +1160,16 @@ export class UserStorage {
     })
   }
 
-  getFullProfile(profileNode): Promise<> {
+  getEncryptedProfile(profileNode): Promise<> {
```

Comment:
Is this method really necessary?

in `getPublicProfile` you're using `loadGunField` directly